auto3dseg/algorithm_templates/dints/scripts/algo.py

```diff
@@ -293,7 +293,7 @@ def objective(trial):
                 cmd += f"--num_images_per_batch {num_images_per_batch} "
                 cmd += f"--num_sw_batch_size {num_sw_batch_size} "
                 cmd += f"--validation_data_device {validation_data_device}"
-                _ = subprocess.run(cmd.split(), check=True)
+                _ = subprocess.run(cmd.split(), env=ps_environ, check=True)
```

Comment:
What is this line for?

Reply:
Hi @dongyang0122 , this is to ensure the environment variables are copied, especially `CUDA_VISIBLE_DEVICES` as commented in L283.